src/WebExternalSearchProvider.cs

```diff
@@ -251,7 +251,7 @@ private IEntityMetadata CreateMetadata(ExecutionContext context, IExternalSearch
 
         private EntityCode GetOriginEntityCode(IExternalSearchQueryResult<WebResult> resultItem, IExternalSearchRequest request)
         {
-            return new EntityCode(EntityType.Organization, this.GetCodeOrigin(), request.EntityMetaData.OriginEntityCode.Value);
+            return new EntityCode(EntityType.Organization, this.GetCodeOrigin(), request.EntityMetaData.OriginEntityCode.Key);
```

Comment:
My interpretation of #36598 is that prior to v4.0.0 the origin entity code was being set correctly

https://github.com/CluedIn-io/CluedIn.Enricher.Web/blob/c42a9ad84a5e7c90f79c7fd6554f630f17b2e4f8/src/WebExternalSearchProvider.cs#L240